spec/db.spec.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../db/index.js');
+const seed = require('../project-utils/seed.js');
+const Bundles = require('../db/models/bundles.model.js');
+require('iconv-lite').encodingExists('foo')
+
+describe('database seeding', () => {
+  test('it should have 100 entries', () => {
+    seed()
+    .then(() => {
+      Bundles.findOne({
+        where: {
+          id: 100,
+        },
+      })
+      .then((results) => {
+        expect(results.dataValues.id).toBe(100);
+        expect(results.dataValues.name).toBe('i go batty for you!');
+        expect(results.dataValues.value).toBe(183);
+        expect(results.dataValues.logoURL).toBe('https://unsplash.com/photos/YcfCXxo7rpc');
+        expect(results.dataValues.helper).toBe('@causam');
+        expect(results.dataValues.minimum).toBe(6);
+      })
+    })
+  });
+});
```

Comment:
This test works great for your seed file, I'm just wondering how the implementation would work if you used a fake data generator. I'll get further clarification in class tonight!

Reply:
See my comment above for a further explanation of why I'm using hard data. Based on your question, I'm a bit confused, are you generating different data each time you seed your database, or is it pulling the same fake data every time you use a fake data generator?